packages/db/postgres/migrations/006-user-used-storage-update.sql

```diff
@@ -0,0 +1,39 @@
+DROP TYPE used_storage
+CREATE TYPE used_storage AS (uploaded TEXT, psa_pinned TEXT);
+
+DROP FUNCTION user_used_storage(bigint);
+
+CREATE OR REPLACE FUNCTION user_used_storage(query_user_id BIGINT) 
+RETURNS used_storage
+LANGUAGE plpgsql
+AS
+$$
+DECLARE used_storage used_storage;
```

Comment:
So, I'd either rename this variable to something different to the type or rename the type to be something more generic like `stored_bytes`.

Reply:
Good point. 
Updated